lib/mcp/server.rb

```diff
@@ -138,6 +140,22 @@ def notify_resources_list_changed
       report_exception(e, { notification: "resources_list_changed" })
     end
 
+    def notify_logging_message(notification_level: nil, logger: nil, data: nil)
+      return unless @transport
+      raise LoggingMessageNotification::NotSpecifiedLevelError unless logging_message_notification&.level
+
+      current_level = notification_level || logging_message_notification.level
+      return unless logging_message_notification.should_notify?(notification_level: current_level)
```

Comment:
`notification` is already clear from the receiver name, so `level` alone might be sufficient.


```suggestion
      return unless logging_message_notification.should_notify?(level: log_level)
```

Reply:
If it is defined as `level` name, I think it will be difficult to distinguish the KW argument name of `should_notify?` from the level instance variable name in `lib/mcp/logging_message_notification.rb`.

As an alternative, how about setting the KW argument name of `should_notify?` to `log_level`?
I think `log_level` is easier to understand because it indicates data received when notifying the log. What do you think?

Reply:
Ah, it doesn't really seem necessary for it to be a keyword argument in the first place. How about making it a positional argument instead?

Reply:
Thank you. You're absolutely right. Fixed.